src/NosCore.GameObject/Character.cs

```diff
@@ -30,10 +32,20 @@ public Character()
             FriendRequestCharacters = new ConcurrentDictionary<long, long>();
             CharacterRelations = new ConcurrentDictionary<long, CharacterRelation>();
             RelationWithCharacter = new ConcurrentDictionary<long, CharacterRelation>();
+            GroupRequestCharacterIds = new List<long>();
+            Group = new Group(GroupType.Group);
```

Comment:
like this group can't be null wich is fine but may need rework on some things :)

Reply:
when you new group you have to add this to the group ;) or the group of the player is not even getting the player